### DISCUSSION TITLE
video demo of obsidian.nvim from OMER HAMERMAN (aka DevOps Toolbox)

### DISCUSSION THREAD

**Discussion by hopezh** [epwalsh/obsidian.nvim#293] [General]
Omer's video demo encouraged me to use obsidian.nvim: [The Holy Grail of Neovim Note Taking
](https://www.youtube.com/watch?v=5ht8NYkU9wQ
)

**Comment by epwalsh** [comment]
It's a great video. Thanks @omerxx! I'm going to highlight this on our README

**Comment by omerxx** [comment]
Thank you both! ❤️
